webrtc.html

```diff
@@ -4946,10 +4947,57 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <code>degradationPreference</code> indicates which is
               preferred.</p>
             </dd>
+            <dt><dfn><code>scalingPolicy</code></dfn> of type
+            <span class="idlMemberType"><a>RTCRtpScalingPolicyType</a></span>,
+            defaulting to <code>"down-only"</code></dt>
+           <dd>
+	     <p> Controls certain aspects of how the browser
+	     can manipulate the media from
+	     a camera or microphone that is sent over RTP by this
+	     <code><a>RtpSender</a></code>.
+	     The aspects this controls are: </p>
+	     <ul>
+	       <li>
+	       <p>changing the width and height of the video frames, </p>
+	       </li>
+	       <li>
+	       <p>changing the same rate of audio, </p>
+	       </li>
+	       <li>
+	       <p>changing the same size of audio. </p>
+	       </li>
+	     </ul>
+          </dd>
           </dl>
         </section>
       </div>
       <div>
+      </div>
+  <pre class="idl">enum RTCRtpScalingPolicyType {
+      "down-only",
+      "up-and-down"
+};</pre>
+        <table data-link-for="RTCRtpScalingPolicyType" data-dfn-for=
+          "RTCRtpScalingPolicyType" class="simple">
+            <tbody>
+              <tr>
+                <th colspan="2">Enumeration description</th>
+              </tr>
+              <tr>
+                <td><dfn><code>"down-only"</code></dfn></td>
+                <td> The browser MUST NOT increase the video width, 
+		  video height, audio sample rate, or the audio sample
+		  size. It MAY decrease any of theses.  </td>
```

Comment:
Would we need to mention somewhere that other things, like frameRate, are _never_ allowed to upscale?